oval_graph/html_builder/graph.py

```diff
@@ -19,13 +19,10 @@ def __init__(self, parts, verbose, all_in_one):
         self.script = self._get_part('script.js')
         self.search_bar = self._get_search_bar()
 
-    def save_html(self, dict_oval_trees, src, rules):
-        self.save_html_report(dict_oval_trees, src)
-        self.print_output_message(src, self._format_rules_output(rules))
-
-    def save_html_report(self, dict_of_rules, src):
+    def save_html(self, dict_oval_trees, src):
         with open(src, "wb+") as data_file:
-            data_file.writelines(self._get_html(dict_of_rules))
+            data_file.writelines(self._get_html(dict_oval_trees))
+        self.print_output_message(src, list(dict_oval_trees.keys()))
```

Comment:
I think that it is better to have the "if verbose" condition here than inside `print_output_message`. The disadvantage of this is that although you call `print_output_message(...)`, nothing is printed if the verbose mode is off, and it is not obvious why.